### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by MariaKonstantinov**
First version of Blog page is ready. Waiting for review. Thanks! :)

### CODE REVIEW

File: blog/blocks/page-content/page-content.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.page-content {
+    max-width: 1152px;
```
Comment (resdenia) [inline blog/blocks/page-content/page-content.css:2]: I think  that part can be simplified it's not so necessary to add that max-width I think  we can add there 858px  and remove from inner content that max-width

File: blog/blocks/page-title/page-title.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.page-title {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 44px;
+    line-height: 1.36;
+    color: #21243d;
+    max-width: 858px;
```
Comment (resdenia) [inline blog/blocks/page-title/page-title.css:7]: if we will add to the page content we can skip that max-width

File: blog/blocks/blogs/blogs.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.blogs {
```
Comment (resdenia) [inline blog/blocks/blogs/blogs.css:1]: if we will add to page-content i think we can skip that file and class!

**Review by resdenia — CHANGES_REQUESTED** [review]
Thank You, Maria! Great work!
Everything is clean and spread to the BEM structure!
Keep doing it!
Please go over my comments and check them.

**Review by meidankaye — APPROVED** [review]

**Pull request merged**